From: Antonio Radici <antonio@debian.org>
Date: Thu, 27 Feb 2014 16:08:26 +0100
Subject: correct_docdir_in_man_page

Fix the link so it points to the correct docdir
and gzipped manual.

Gbp-Pq: Topic debian-specific
---
 doc/mutt.man | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/mutt.man b/doc/mutt.man
index 181ef62..718f87a 100644
--- a/doc/mutt.man
+++ b/doc/mutt.man
@@ -185,7 +185,7 @@ User's personal mapping between MIME types and file extensions.
 System mapping between MIME types and file extensions.
 .IP "@bindir@/mutt_dotlock"
 The privileged dotlocking program.
-.IP "@docdir@/manual.txt"
+.IP "/usr/share/doc/mutt/manual.txt.gz"
 The Mutt manual.
 .SH BUGS
 .PP
